package_scripts/subset-fonts.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * The fundamentals for this utility are derived from
+ * https://stackoverflow.com/questions/64614572/creating-a-material-icons-subset
+ */
+
+import * as fs from "fs"
+import * as path from "path"
+import { promisify } from "util"
+import { exec } from "child_process"
```

Comment:
We have [exec-sh](https://www.npmjs.com/package/exec-sh) in the package.json already, which might be more convenient.